frontend/src/util/LiveMessageManager.js

```diff
@@ -0,0 +1,84 @@
+import {
+  ROLE
+} from 'tracim_frontend_lib'
+
+export const LIVE_MESSAGE_STATUS = {
+  OPEN: 'open',
+  CLOSE: 'close'
+}
+
+export class LiveMessageManager {
+  constructor() {
+    this.status = LIVE_MESSAGE_STATUS.CLOSE
+  }
+
+  openLiveMessageConnection () {
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        console.log('resolving openLiveMessageConnection')
+
+        // this.mockRandomLiveEvents()
+
+        this.status = LIVE_MESSAGE_STATUS.OPEN
+
+        resolve(true)
+      }, 200)
```

Comment:
Why 200? It seems arbitrary. Is there no other way than a timeout of 200 ms?

Maybe put this value in a constant?

Reply:
it is arbitrary, this code is still work in progress for the POC, 200 is for debug test.
I should have mentioned it sry